setup.py

```diff
@@ -763,9 +763,12 @@ def add_multiarch_paths(self):
         tmpfile = os.path.join(self.build_temp, 'multiarch')
         if not os.path.exists(self.build_temp):
             os.makedirs(self.build_temp)
-        ret = os.system(
-            'dpkg-architecture -qDEB_HOST_MULTIARCH > %s 2> /dev/null' %
-            tmpfile)
+        devnull = open(os.devnull, 'wb')
```

Comment:
I don't think there's really a point wiring up stderr to /dev/null, as it's just as effective to ignore it by not passing in a pipe to the stderr parameter.